connection_test.go

```diff
@@ -351,19 +369,21 @@ func (h onErrorTestHandler) OnError(ctx context.Context, err error) {
 }
 
 func TestTimeout(t *testing.T) {
-	WithVerifiedServer(t, nil, func(ch *Channel, hostPort string) {
+	// TODO: Add relaying.
```

Comment:
change todo to mention that we need to implement timeouts in the relay before we can enable this test